assets/dev/scss/editor/panel/controls/_repeater.scss

```diff
@@ -17,6 +17,14 @@
 		}
 	}
 
+	&.elementor-repeater-has-maximum-rows {
```

Comment:
i'm not sure about the name of the CSS class 
maybe in the future we will want to hide the button for another reason

Reply:
@TzviRabinovitch this class is hiding both the add more button and the button to copy the row, so it basically handles everything that needs to change when the max number of rows is reached.
I could refactor this and add two classes ( elementor-repeater-hide-add-button and elementor-repeater-hide-copy-button ) so that the code could be reused in the future when you want to hide the button. What do you think?